src/dynatrace/opentelemetry/metrics/export/_exporter.py

```diff
@@ -45,17 +50,18 @@ class _DynatraceMetricsExporter(MetricExporter):
     """
 
     def __init__(
-            self,
-            endpoint_url: Optional[str] = None,
-            api_token: Optional[str] = None,
-            prefix: Optional[str] = None,
-            default_dimensions: Optional[Mapping[str, str]] = None,
-            export_dynatrace_metadata: Optional[bool] = False,
-            preferred_temporality: Dict[type, AggregationTemporality] = None,
-            preferred_aggregation: Dict[
-                type, Aggregation
-            ] = None,
+        self,
+        endpoint_url: Optional[str] = None,
+        api_token: Optional[str] = None,
+        prefix: Optional[str] = None,
+        default_dimensions: Optional[Mapping[str, str]] = None,
+        export_dynatrace_metadata: Optional[bool] = False,
+        preferred_temporality: Dict[type, AggregationTemporality] = None,
```

Comment:
should we drop this and always set `_DYNATRACE_TEMPORALITY_PREFERENCE`? :thinking:

Reply:
+1. I think we should not give the option to pass a different temporality and "hard-code" it within the exporter.

Reply:
I moved it into the constructor - that also allowed me to "hide" `_DYNATRACE_TEMPORALITY_PREFERENCE`, since it's only used in the exporter now. I deleted the _constants file again, which was only there because I needed access to that variable from multiple places.